kuma/urls.py

```diff
@@ -19,23 +14,21 @@
 admin.autodiscover()
 
 urlpatterns = [re_path("", include("kuma.health.urls"))]
-# The non-locale-based landing URL's
-urlpatterns += [re_path("", include("kuma.landing.urls"))]
-urlpatterns += i18n_patterns(
-    re_path(
-        r"^events",
+urlpatterns += [
+    path(
+        "<locale>/events/",
```

Comment:
We need to remove the trailing slash, or if we don't, we'll need to add `events` to:

 https://github.com/mdn/yari/blob/c8c5860bf9331690a3fb17874c663d7eafea0add/deployer/aws-lambda/content-origin-request/index.js#L28.

As it is, it works locally, but not when deployed to stage. On stage, a request for `/en-US/events` is redirected to `/en-US/events/`, but then that request is redirected back to `/en-US/events` by the content-origin-request lambda code:

https://github.com/mdn/yari/blob/c8c5860bf9331690a3fb17874c663d7eafea0add/deployer/aws-lambda/content-origin-request/index.js#L182

```suggestion
        "<locale>/events",
```

Reply:
@peterbe On second thought, I think a better way to handle this `events` redirect is via the fundamental redirects, which I've gone ahead and done in https://github.com/mdn/yari/pull/4234. So let's just remove this endpoint (and its tests) altogether from Kuma.

Reply:
I don't think so. Now, with regular `path()` we get the best of both worlds. 
A GET to `/en-US/events` is a redirect to `https://community.mozilla.org/events/` but that's because it does **2 redirects**. 
```
▶ xh http://localhost:8000/en-US/events
HTTP/1.1 301 Moved Permanently
location: /en-US/events/

▶ xh http://localhost:8000/en-US/events/
HTTP/1.1 302 Found
location: https://community.mozilla.org/events/
```

So the net effect is that our URL works with or without a slash. 

I think the code here in `urls.py` is correct but I'll need to re-read your comment about the headless tests.

Reply:
Oh I think I see now. What Lambda does matters. 
The big question is; can't we just move this redirect to the fundamental redirects instead of even having any of it here in Kuma??

Reply:
If you can merge https://github.com/mdn/yari/pull/4269 we can totally rip out all of this code from kuma.